### DISCUSSION TITLE
The same table is distributed on two pages, and some data extraction fails

### DISCUSSION THREAD

**Discussion by AresElvis** [jsvine/pdfplumber#725] [Ask for help with specific PDFs]
## Describe the bug

The table on the first page is partly on the second page due to space reasons, but the data in the first row of the second page cannot be extracted during extraction

## Code to reproduce the problem

file_name = '《新能源汽车推广应用推荐车型目录》（2022年第6批）车型主要参数.pdf.pdf'
pdf = pdfplumber.open(file_path_dir + file_name)
for i, page in enumerate(pdf.pages):
    tables = page.extract_tables()

## PDF file

[《新能源汽车推广应用推荐车型目录》（2022年第6批）车型主要参数.pdf.pdf](https://github.com/jsvine/pdfplumber/files/9443100/2022.6.pdf.pdf)

## Expected behavior

外廓尺寸宽(mm): 2550
<img width="937" alt="image" src="https://user-images.githubusercontent.com/33786487/187163389-a00f66ba-b4ab-488d-af24-fa5b7ed1aa7e.png">

## Actual behavior

<img width="584" alt="image" src="https://user-images.githubusercontent.com/33786487/187163156-5b00d601-fc0d-46b6-9265-0e5d96fd23d1.png">

## Screenshots

*If applicable, add screenshots to help explain your problem.*

## Environment

- pdfplumber version: [e.g., 0.7.1]
- Python version: [e.g., 3.8.1]
- OS: [e.g., Mac, ]

## Additional context

*Add any other context/notes about the problem here.*

**Comment by jsvine** [comment]
Hi @AresElvis, and thanks for your interest in this library. Your inquiry is more of a request for troubleshooting with a specific PDF rather than a bug in the library, so I'm moving it here. You might find this prior discussion helpful: https://github.com/jsvine/pdfplumber/discussions/625

Let me know if that guidance works for you. If not, let me know where you're encountering trouble 👍
